css-technical-plan.md

```diff
@@ -0,0 +1,131 @@
+# Atlas (CSS) tech plan
+
+## Links
+
+- [Current version of tokens](https://www.figma.com/proto/03HGueCJEx4tyZd3Rhfw5B/DevRel-Design-System?node-id=268%3A343&viewport=1143%2C-507%2C0.08135364204645157&scaling=scale-down-width)
+- [Colors](https://www.figma.com/file/6ZQIcp2AS8022uU96v0xx4/ATLAS---COLOR?node-id=0%3A1)
+
+## Task list
+
+👨‍💻 Constitutes a developer task
+💭 Pondering, not ready, but input welcome
+⚜ Group or PM task
+
+- fill out all Open Source related files ⚜
+  - update project README.md ⚜
+  - 👨‍💻 write initial setup steps
+  - 👨‍💻 write steps for inclusion in a project
+  - 👨‍💻 update css folder README.md
+  - ⚜ review and update CODE_OF_CONDUCT.md (group task / PM task?)
+- 👨‍💻 add monorepo related files
+  - add top level index file scss in /css
+  - add top level package.json and init monorepo
+- 👨‍💻 implment changesets [changesets](https://github.com/atlassian/changesets)
+- 👨‍💻 implement tokens (each direct sub list item is a dev task)
+  - color tokens
+    - convert tokens to hsla (pending dropping IE theming confirmation)
+    - raw colors in palette.scss
+    - themed colors in color-themes.scss
+  - 👨‍💻 spacing tokens
+  - 👨‍💻 typography, font-size tokens
+  - 👨‍💻 depth/shadow tokens
```

Comment:
For future consideration: z-index tokens

Reply:
We have some already in docs-ui, I see no reason why they can't move initially.

Reply:
They look like this:

```
// State related z-index
$zindex-active: 1;
$zindex-hover: 2;
$zindex-focus: 3;
$zindex-multi: 4;
// Component type related z-index
$zindex-dropdown: 1000;
$zindex-sticky: 1020;
$zindex-fixed: 1030;
$zindex-modal-backdrop: 1040;
$zindex-modal: 1050;
$zindex-popover: 1060;
$zindex-tooltip: 1070;